mod_auth/forms.py

```diff
@@ -2,45 +2,48 @@
 ccextractor-web | mod_auth/forms.py
 
 Author   : Saurabh Shrivastava
-Email    : saurabh.shrivastava54@gmail.com
-Link     : https://github.com/saurabhshr
+Email    : saurabh.shrivastava54+ccextractorweb[at]gmail.com
+Link     : https://github.com/saurabhshri
 
 Based on https://github.com/CCExtractor/sample-platform/blob/master/mod_auth/forms.py
-Author   : Willem Van Iseghem
-Email    : github@canihavesome.coffee
-Link     : https://github.com/canihavesomecoffee
 """
 
 from flask_wtf import FlaskForm
 from wtforms import PasswordField, StringField, SubmitField
 from wtforms.fields.html5 import EmailField
 from wtforms.validators import DataRequired, Email, ValidationError
+from email_validator import validate_email, EmailNotValidError
+
 
 def valid_password(form, field):
     pass_size = len(field.data)
     if pass_size == 0:
         raise ValidationError('The password can not be empty!')
-    if pass_size < 8 or pass_size > 50:
+    if pass_size < 8 or pass_size > 128:
```

Comment:
I'd extract both the min & max length into a variable and then also put those in the error message. Easier to maintain.

Reply:
Added.